metasvc-model/src/main/java/de/digitalcollections/model/identifiable/entity/manifestation/Manifestation.java

```diff
@@ -409,11 +409,12 @@ public int hashCode() {
    * @param relations a list of EntityRelations
    * @return A texual representation of the list with subject uuid, predicate and object uuid
    */
-  private String dumpShortenedRelations(List<EntityRelation> relations) {
+  public static String dumpShortenedRelations(List<EntityRelation> relations) {
     return "["
         + (relations == null
             ? "null"
             : relations.stream()
+                .filter(Objects::nonNull)
```

Comment:
In dieser Liste sollten wirklich keine `null`s vorkommen. Machst Du dafür bitte noch ein Ticket? Das sollten wir uns mal anschauen.

Reply:
Ticket DFGMETA-71 ist angelegt.